File: app/src/main/java/io/twere/plainpure/data/model/dribbble/realm/LikeRealm.java
package io.twere.plainpure.data.model.dribbble.realm;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;
import java.util.Date;

@RealmClass public class LikeRealm implements RealmModel {

  @PrimaryKey  private long id;

  private Date createdAt;

  private UserRealm user;

  private ShotRealm shot;

  public long getId() {
    return id;
  }

  public void setId(long id) {
    this.id = id;
  }

  public Date getCreatedAt() {
    return createdAt;
  }

  public void setCreatedAt(Date createdAt) {
    this.createdAt = createdAt;
  }

  public UserRealm getUser() {
    return user;
  }

  public void setUser(UserRealm user) {
    this.user = user;
  }

  public ShotRealm getShot() {
    return shot;
  }

  public void setShot(ShotRealm shot) {
    this.shot = shot;
  }
}

File: app/src/main/kotlin/io/twere/plainpure/data/local/like/LikeRealm.kt
package io.twere.plainpure.data.local.like

import io.twere.plainpure.data.local.shot.ShotRealm
import io.twere.plainpure.data.local.user.UserRealm
import io.realm.RealmModel
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import java.util.Date

@RealmClass
open class LikeRealm : RealmModel {

  @PrimaryKey
 open var id: Long = 0
 open var createdAt: Date? = null
 open var user: UserRealm? = null
 open var shot: ShotRealm? = null

}